superduperdb/vector_search/in_memory.py

```diff
@@ -64,6 +67,14 @@ def find_nearest_from_id(self, _id, n=100):
         return self.find_nearest_from_array(self.h[self.lookup[_id]], n=n)
 
     def find_nearest_from_array(self, h, n=100, within_ids=None):
+        if self.h is None:
+            logging.error(
+                'Tried to search on an empty vector database',
+                'Vectors are not yet loaded in vector database.',
+                '\nPlease check if model outputs are ready.',
+            )
+            return [], []
+
         self.post_create()
```

Comment:
Move the condition of `self.h` below `self.post_create`

Reply:
yes done